### PR TITLE
Prepare initial third party services

### PR DESCRIPTION
**Pull request by Fryguy**
Prepare initial third party services

### CODE REVIEW

**Comment by Fryguy** [comment]
@kbrock Please review.

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,28 @@
 ---
-sudo: false
```
Comment (kbrock) [inline .travis.yml]: 😮

**Comment by kbrock** [comment]
@Fryguy this looks good to me. ok to merge?

I am have a branch with a couple of easy fixes for code climate

**Comment by kbrock** [comment]
> pull requests that have a [useless] failing status can’t be merged on a phone

Sigh

Will merge next week if you don’t first @fryguy

**Comment by kbrock** [comment]
@Fryguy Can you add config?

```
Linter/Yaml - missing config files
```

**Pull request merged**